src/source/release-notes.rst

```diff
@@ -0,0 +1,194 @@
+
+.. list-table::
+ :widths: 20 80
+ :header-rows: 1
+
+ * - Field
+   - Change synopsis
+ * - cpc 		            
+   - moved to bidMicros. Now allows for 'AUTOMATIC' too.
+ * - totalBudget            
+   - moved to budgets.total.limitMicros. Now allows 'UNLIMITED' too.
+ * - spentBudget               
+   - moved to budgets.total.spentMicros
+ * - dailyBudget            
+   - moved to budgets.daily.limitMicros. Now allows 'UNLIMITED' too.
+ * - dailySpent            
+   - moved to budgets.daily.spentMicros
+ * - campaignId          
+   - new, not yet mandatory.
+ * - status             
+   - can now contain "DOMAIN_PENDING", requiring the seller to confirm the new clickout url domain before the can go active.
+ * - price                
+   - moved to price.amountCents
+ * - priceType            
+   - moved to price.priceType
+ * - priceUnit            
+   - moved to price.priceUnit
+ * - originalPrice       
+   - moved to price.originalAmountCents
+ * - currency             
+   - removed
+ * - externalId           
+   - removed
+ * - statusReasons        
+   - removed, status is expanded
+ * - pageNumber           
+   - removed
+ * - suggestedCpcForPageOne
+   - removed
+ * - allowPaypal            
+   - removed
+
+
+Categories
+""""""""""
+
+See :ref:`categories` for full details.
+
+.. list-table::
+ :widths: 20 80
+ :header-rows: 1
+
+ * - Field
+   - Change synopsis
+ * - currency               
+   - removed
+ * - cpc
+   - moved to bidMicros.
+ * - totalBudget            
+   - moved to totalBudgetMicros.
+ * - minDailyBudget
+   - moved to dailyBudgetMicros. This is now an interval instead of min value.
+ * - paypalEnabled
+   - removed
+ * - paypalBpEnabled
+   - removed
+ * - suggestedCpcForPageOne
+   - removed
+
+Campaigns
+"""""""""
+Added first campaign support. See :ref:`campaigns` for full details.
+
+* :ref:`get_campaign` to get list of campaigns for a user. Expected 0 or 1 result.
+* :ref:`get_campaign_id` to fetch the individual campaign.
+* :ref:`post_campaign` to create a campaign. Currently only 1 allowed per seller.
+* :ref:`put_campaign_id` to update the campaign. Currently only status & budgets can be updated.
+* :ref:`put_campaign_id_status` to manage the campaign status (ACTIVE,PAUSED) directly.
+* :ref:`put_campaign_id_budgets` to manage the campaign's budgets directly.
+
+.. note::
+ For the time being, we will create a campaign underwater for users who start placing ads without first having created a campaign. 
+ Be sure to check the budgets of that campaign as a default campaign will have unlimited budget.
+ Deleting a campaign is currently not allowed.
+
+User
+""""
+
+See :ref:`get_user` for full details.
+
+Start using V4. There are 2 new fields in comparison with V2:
+``hasAds`` (**bool**), whether a user has ads in the system and 
+``isApiManaged`` (**bool**), whether we see API partners managing ads for the user (not true if an api partner is using RO scopes only)
```

Comment:
I'm not sure why we decided to expose it in the end :-/ it was supposed to be a temporary thing, given the implementation is based also on "proxy" data. It could be that the user is API managed but not much activity in the past (90?) days so we see no active tokens

Reply:
if we want to have only 1 user version available, we need to be able to explain all the fields. I'd also rather remove these two fields, but 
1) console needs it 
2) BNL wants to keep the functionality around `isAPiManaged` since it's about an upsell message 😭